src/bdc/pipeline.py

```diff
@@ -1,185 +1,185 @@
-# SPDX-License-Identifier: MIT
```

Comment:
Can not we then just delete this class and rename "pipeline_dal" to "pipeline" ?

Reply:
maybe sum_insights_database? I agree it doesn;t necessarily need to be called DAL, even though that's what it is. I'll change it now.

With replacing the pipeline, I'm going to leave it there for this merge because the DAL is still new, but we will need to go through and delete all unused code/demos at some point.